packages/netsuite-adapter/src/filters/element_references.ts

```diff
@@ -135,9 +135,11 @@ const getServiceElemIDsFromPaths = (
 ): ElemID[] =>
   foundReferences
     .flatMap(ref => {
-      const absolutePath = pathPrefixRegex.test(ref)
-        ? resolveRelativePath(element.value[PATH], ref)
-        : FILE_CABINET_PATH_SEPARATOR.concat(ref)
+      const absolutePath = resolveRelativePath(element.value[PATH], ref)
+      // TODO: The log should be removed when SALTO-4025 is communicated.
+      if (!pathPrefixRegex.test(ref)) {
+        log.debug('Found a possible file reference without a path prefix: %s', ref)
```

Comment:
to keep the original behavior before this [commit](https://github.com/salto-io/salto/commit/a05213074a6569dc4ebe217abfdd5e3094d69be8), and to log only found references:
```suggestion
        const maybeServiceIdRecord = serviceIdToElemID[FILE_CABINET_PATH_SEPARATOR.concat(ref)]
        if (_.isPlainObject(maybeServiceIdRecord)) {
          log.debug('Found a possible file reference without a path prefix: %s', ref)
        }
        return [ref]
```